content/blog/2021-06-25-june-21-community-gems.md

```diff
@@ -0,0 +1,121 @@
+---
+title: June '21 Community Gems
+date: 2021-06-25
+description: |
+  A roundup of technical Q&A's from the DVC community.
+  This month: remote storage integration, hyperparameter tuning,
+  best practices for managing experiments and more.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community. 
+  This month: remote storage integration, hyperparameter tuning,
+  best practices for managing experiments and more.
+picture: 2021-05-25/gems-cover.png
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/june-21-community-gems/779
+tags:
+  - Community
+  - Pipelines
+  - Remote Storage
+---
+
+### [Q: Is it possible to plot multiple experiments together?](https://discord.com/channels/485586884165107732/563406153334128681/834387923482181653)
+
+You can use experiment names in the `dvc plots` commands. You need to use the
+`diff` command to compare multiple plots. Try
+`dvc plots diff exp-6ef18 exp-b17b4 exp-26e88`.
+
+Thanks to @PythonF from Discord for asking this question that led to this Gem!
+💎
+
+### [Q: Where are the configs for an experiment being pushed in Git when I run `dvc exp push`?](https://discord.com/channels/485586884165107732/563406153334128681/837773937390649364)
+
+It uses custom Git refs internally, similar to the way GitHub handles PRs. It’s
+a custom DVC Git ref pointing to a Git commit. Here's an example.
+
+```bash
+$ git show-ref exp-26220
+c42f48168830148b946f6a75d1bdbb25cda46f35 refs/exps/f1/37703af59ba1b80e77505a762335805d05d212/exp-26220
+```
+
+You can read more about how we handle our custom Git refs in
+[this blog post](https://dvc.org/blog/experiment-refs).
+
+Thanks to @Chandana for asking this question about experiments!
+
+### [Q: Is there a way to list all the experiments I have on my DVC remote that have not been committed to Git?](https://discord.com/channels/485586884165107732/563406153334128681/836705209039978538)
+
+Yes! The command `dvc exp list gitremote --all` lists all of the experiments for
+the default remote in the Git repo.
+
+Thanks again @Chandana for this gem!
+
+### [Q: Is CML compatible with Azure DevOps?](https://discord.com/channels/485586884165107732/728693131557732403/841664412221177926)
+
+Another great question from @Chandana!
+
+Right now, we support GitHub and GitLab.
+
+Azure DevOps and GCP (Google Cloud Platform) support are on the roadmap. Stay
```

Comment:
Actually we could use two tickets from CML

Reply:
You want me to make the tickets for Azure and GCP?

Reply:
This one https://github.com/iterative/terraform-provider-iterative/issues/64 ?

Also GCP and Azure DevOps are two conceptually different things Azure DevOps is an alternative to GH Actions, GitLab CI, Jenkins, etc (that's why ticket is in the CML repo)

GCP is similar to AWS, Azure support to provision instances to run your training (that's why ticket is in the terraform provider repo).

May be GCP also has some thing similar to Azure pipelines or something like that cc @iterative/cml ?

Reply:
Otherwise I would not mix those here.

Reply:
We [plan](https://github.com/iterative/terraform-provider-iterative/pull/64) to support [Google Cloud Compute Engine](https://cloud.google.com/compute) pretty soon, but [Google Cloud Build](https://cloud.google.com/build) is not even in the roadmap. The latter would be the direct equivalent of [Azure DevOps](https://azure.microsoft.com/en-us/services/devops).

I'd keep them separate or, at least, clarify the differences between all the services we support or plan to.

* GitLab and GitHub are all-in-one development services offering version control, CI/CD and collaboration features. 
* Azure DevOps is just a CI/CD platform, which is suspiciously similar to GitHub Actions. 😉
* Google Cloud Platform is a whole public cloud that we only plan to use to provide GPU cloud computing support on CML, DVC, Studio...